recipes/tinyspline/all/test_package/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+if(TINYSPLINE_API_0_3)
+  add_definitions(-DTINYSPLINE_API_0_3)
```

Comment:
cmake_find_package provides package's version. You could use TINYSPLINE_VERSION directly

Reply:
Indeed, but I've not added cmake_find_package testing because conan's cmake_find_package generator support is not yet complete.
tinyspline requires the cmake_find_package_multi generator to create 2 files: `tinyspline-config.cmake` and `tinysplinecxx-config.cmake`.
There is a `FIXME` in package_info and I've not added any cmake_find_package info to package_info.